src/aat/resources/features/F-1009 - Submit Case Creation Handle Case Links/F-1009_Test_Data_Citizen_Base_Error_Response.td.json

```diff
@@ -0,0 +1,26 @@
+{
+	"_guid_": "F-1009_Test_Data_Citizen_Base_Error_Response",
+
+	"productName": "CCD Data Store",
+	"operationName": "Submit case creation as Citizen",
+
+	"method": "POST",
+	"uri": "/citizens/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases",
```

Comment:
If a **common TD file** to cover the basics of the API signature does not exist can we create on and use it as this endpoint is being used multiple times in these tests:

* F-1009_Test_Data_Citizen_Base
* F-1009_Test_Data_Citizen_Base_Error_Response

Reply:
Done [c129a9a](https://github.com/hmcts/ccd-data-store-api/pull/1872/commits/c129a9a6a6fdff4226e65e0b027691d3cd49352f)